### ISSUE TITLE
Feature request: illuminate, overlap between a.m./p.m. and numbers in time row

### ISSUE THREAD

**Issue by ellenbroadman** [Artificial-Pancreas/iAPS#277]
when I increase the text size on the homepage, it is hard to read the timeline because the numbers overlap with a.m. and p.m. One solution is to use A for a.m. and P for p.m.. Thank you for considering fixes.

![image](https://github.com/Artificial-Pancreas/iAPS/assets/96346244/76a1101c-9276-467e-bd92-53ad1a39c26b)

**Comment by Jon-b-m** [comment]
I think forcing it to always use 24hour format would look better. Do you agree?

**Comment by ellenbroadman** [comment]
Good idea.  makes that line more simple.  either approach seems fine to me

**Comment by ellenbroadman** [comment]
how about using 2 rather than 2:00.  Some people might prefer that so they do not have to figure out that 15 is 3p.  And would make for a simple line without overlap.  I am fine with the 24 hour format and agree that would be the most simple visually (although not mentally for some).







;

**Comment by Jon-b-m** [comment]
implemented. Please update your iAPS version
